src/__tests__/LogoNav.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { shallow } from "enzyme";
+import React from "react";
+import Logo from "../components/LogoNav";
+
+describe("rendering Typography", () => {
```

Comment:
Maybe you can create a "theme" folder with a  "__tests__" folder in src, then move all themes components and test files into it :)